tests/runners/dataset_runner.py

```diff
@@ -31,20 +35,19 @@ def valid_run(self, dataset_fixture, project_uuid=None):
         self.submission_manager.stage_data_files(self.dataset.config['data_files_upload_area_uuid'])
         self.submission_manager.wait_for_envelope_to_be_validated()
 
-    def archived_run(self, dataset_fixture):
-        self.valid_run(dataset_fixture)
-        self.submission_manager.submit_envelope(["Archive"])
-        self.submission_manager.wait_for_envelope_to_be_archiving()
+    def direct_archived_run(self, dataset_fixture):
```

Comment:
Might not be bad to retain running the current test with DSP since we haven't totally moved to direct archiving yet.

Reply:
@aaclan-ebi It is still there on line 46 :-)